region/lowernorfair/east.json

```diff
@@ -131,6 +142,22 @@
                   ],
                   "note": "Take the platforms two at a time by building some run speed for each jump on a one tile runway."
                 },
+                {
+                  "name": "Power Bomb Clear",
+                  "notable": false,
+                  "requires": [
+                    "h_canNavigateHeatRooms",
+                    "h_canUsePowerBombs",
```

Comment:
This could be replaced with `Morph`. This helper uses 1 ammo which shouldnt be necessary